recipes/toad/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "toad" %}
+{% set version = "0.0.58" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9dce218ac11eb9e3a80b233557694d0d349418a1ba517306919dff02595e5f07
+
+build:
+  number: 0
+  skip: True  # [py==38 or py==35 or py==27]
```

Comment:
Since toad is >=3.5 you can remove `py==35` from here (I am sorry if I accidentally added it earlier). Out of curiosity, can you please try omitting `py==38` as well?

Reply:
I have removed py35 and py38, it seems runs well.